endpoints/openrtb2/firstpartydata/first_party_data.go

```diff
@@ -0,0 +1,238 @@
+package firstpartydata
+
+import (
+	"encoding/json"
+	"errors"
+	"github.com/buger/jsonparser"
+	"github.com/evanphx/json-patch"
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"github.com/prebid/prebid-server/util/jsonutil"
+)
+
+const (
+	site = "site"
+	app  = "app"
+	user = "user"
+	data = "data"
+)
+
+func GetFPDData(request []byte) ([]byte, map[string][]byte, error) {
```

Comment:
If this function just works with global FPD data I think it would be good to have a function name that makes that clear.

Reply:
Renamed